packages/form/src/components/TextInputFieldV2/__stories__/Template.stories.tsx

```diff
@@ -0,0 +1,19 @@
+import type { StoryFn } from '@storybook/react'
+import { Stack } from '@ultraviolet/ui'
+import type { ComponentProps } from 'react'
+import { TextInputField } from '..'
+import { Submit } from '../..'
+
+export const Template: StoryFn<
+  ComponentProps<typeof TextInputField>
+> = args => (
+  <Stack gap="1">
```

Comment:
```suggestion
  <Stack gap={1}>
```

?

Reply:
@vincentaudebert Look like gap can be a `number` or a key of space in the `lightTheme`

Reply:
@vincentaudebert we don't have restrictions onto that both are being used